src/modules/timeAgo/index.ts

```diff
@@ -0,0 +1,6 @@
+import TimeAgo from "./timeAgo";
+
+export default (datetime: string): string => {
+	const timeago = new TimeAgo(datetime);
```

Comment:
The API should be something like this:

```
const current = timeAgo();
```
if we pass no param, it should determine the current date-time